src/extensions/markdown/Link/plugins/LinkTooltipPlugin/index.tsx

```diff
@@ -111,14 +111,12 @@ class SelectionTooltip implements PluginView {
         if (prevState && prevState.doc.eq(state.doc) && prevState.selection.eq(state.selection))
             return;
 
-        const prevTextNode = this.textNode;
         this.textNode = getTextNode(view.state);
 
         const prevRef = this.textNodeRef;
         this.updateTextNodeRef();
 
         if (!this.textNode || !this.textNodeRef) {
-            this.removePlaceholderLink(prevTextNode);
```

Comment:
1. Select the word
2. Click the link button
3. A popup will appear
4. Click somewhere on the document

Previously
the link was deleted because the text was not entered in the popup

Now
the link is not deleted

You should not change this behavior.